component/go.mod

```diff
@@ -35,6 +35,8 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
+
+
```

Comment:
Was this added by a tool?

Reply:
this was added in the previous PR by crosslink removing other dependencies and leaving the new lines until i re-ran the tool